src/runtime/components.ts

```diff
@@ -0,0 +1,6 @@
+import { defineNuxtPlugin } from '#imports'
+import CreateAnimation from './components/CreateAnimation.vue'
+
+export default defineNuxtPlugin(nuxtApp => {
+  nuxtApp.vueApp.component('CreateAnimation', CreateAnimation)
```

Comment:
> Instead of registering it globally, what about using the `addComponent` utility from `@nuxt/kit` to auto import only where it is used?

Good call, I'll change to nuxt kit's utility 👍🏽 

> And is there any merit in using another, clearer name, like `IonAnimation` - or is that confusing?

I thought it would be intuitive since it works with ionic's `createAnimation` function, no?

Reply:
I think `<IonAnimation>` might be better (or another name?). I wouldn't expect a component to have a method-style name. (e.g. `useHead({ meta: [] })` becomes `<Meta>`). And we can document it.